01/ht/gendos379/02/mybind-script.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Creates a new function that, when called, 
+ * has its this keyword set to the provided value.
+ */
+Function.prototype.myBind = function() {
+    var func = this;
+    var args = Array.prototype.slice.call(arguments);

```

Comment:
you mean, that i can change `Array.prototype` to `[]`, so the line will look like this `var args = [].slice.call(arguments);` ?